modules/apps/29-fee/types/ack.go

```diff
@@ -0,0 +1,22 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// NewIncentivizedAcknowledgement creates a new instance of IncentivizedAcknowledgement
+func NewIncentivizedAcknowledgement(relayer string, ack []byte) IncentivizedAcknowledgement {
+	return IncentivizedAcknowledgement{
+		Result:                ack,
+		ForwardRelayerAddress: relayer,
+	}
+}
+
```

Comment:
You also need to add a `Success` method here. Just return `false` if `ForwardRelayerAddress` is empty, and `true` otherwise